src/script/hooks/useResizeObserver/useResizeObserver.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Wire
+ * Copyright (C) 2025 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
```

Comment:
We already had a useResizeObserver in MessageList.tsx. Did you replace that one, or do we have the same function name two times now?

Reply:
Didn't see it, I'm gonna rename my version, like useElementSize, because it's different the already existing `useResizeObserver`